client/src/App.js

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import { MuiThemeProvider } from '@material-ui/core';
 import { BrowserRouter, Route, Switch, Redirect } from 'react-router-dom';
 
-import { theme } from './themes/theme';
-import { UserContext } from './components/UserContext';
+import { theme } from 'themes/theme';
+import { UserContext } from 'components/UserContext';
 
-import NavBar from './components/NavBar';
-import LandingPage from './pages/Landing/Landing';
+import NavBar from 'components/NavBar';
```

Comment:
```suggestion
import NavBar from './components/NavBar';
```

Reply:
Since my last merge we can use absolute imports

Reply:
Actually since THIS merge